client/src/components/App.jsx

```diff
@@ -21,15 +22,14 @@ const App = () => {
       });
   }
 
-
   const [siteData, setSiteData] = useState(null);
   const [modalOn, setModalOn] = useState(false);
   const [currentPicIndex, setCurrentPicIndex] = useState(0);
 
   const incrementHelpful = () => {
     let newSiteData = siteData;
     newSiteData.pictures[currentPicIndex].helpful = newSiteData.pictures[currentPicIndex].helpful + 1;
-    incrementHelpfulInDB(siteData.siteID, newSiteData.pictures);
+    pushHelpfulToServer(siteData.siteID, newSiteData.pictures);
```

Comment:
This seems like a useful change, wondering if push is the right word since it has such a specific meaning?

Reply:
I wonder about this too... was thinking about a name for a while and that was the best I could come up with right now.